### PR TITLE
fix(binance) - message hashes

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/pro/binance.ts [hunk 1]
```diff
@@ -906,6 +904,14 @@ export default class binance extends binanceRest {
         }
     }
 
+    messageHash (unifiedChannel: Str, symbol: Str = undefined) {
```
Comment (carlosmiei) [inline ts/src/pro/binance.ts:907]: can we remove this method?

Comment (ttodua) [reply ts/src/pro/binance.ts]: ah was remnant. now removed

**Comment by ttodua** [comment]
c# tests fail as always, when tested locally, it passed all langs

**Comment by carlosmiei** [comment]
> c# tests fail as always, when tested locally, it passed all langs

@ttodua we have to check what is causing the error in c#

**Pull request merged**